docs/installationguide/ubuntu.rst

```diff
@@ -224,7 +224,7 @@ We also need to install php5-gd (as of 1.28.107, this is not installed)
 
 **Step 6**: Edit Timezone in PHP
 
-vi /etc/php5/apache2/php.ini
+``sudo nano /etc/php5/apache2/php.ini``
```

Comment:
Would rather this be changed back.  The `sudo` change makes sense, as `sudo` is used elsewhere in the doc.  I'd rather not have default editors changed, though.